run-emulator.sh

```diff
@@ -18,6 +18,27 @@ if [ x"$GDBSERVER" != x"" ]; then
     TAIL_ARGS="$TAIL_ARGS -s -S"
 fi
 
+dns_servers=""
+if [ x"$EMU_DNS_SERVER" != x"" ]; then
+    dns_servers=$EMU_DNS_SERVER
+fi
+
+# DNS servers from command line arg override ones from environment variable.
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -dns-server)
```

Comment:
Please use '--dns-server' as parameter name.

The convention on Unix is single dash for single-case names, and double dash for long names.